### PR TITLE
[update] resolves #56

### LINKED ISSUE #56: API for customizing page number labels
Some books start their page numbers with Roman Numerals and then 1, 2, 3... API is requested to make this customization possible so that users won't be confused for page numbers not matching correctly.

### PR DESCRIPTION
**Pull request by ZhijieZhang**

### CODE REVIEW

File: src/components/PageNavOverlay/PageNavOverlay.js [hunk 1]
```diff
@@ -16,19 +16,31 @@ class PageNavOverlay extends React.PureComponent {
...
-    if (prevProps.currentPage !== this.props.currentPage) {
-      this.setState({ input: this.props.currentPage });
+    const { currentPage, pageLabels } = this.props;
+    
+    const isCustomPageLabels = prevProps.pageLabels !== this.props.pageLabels && prevProps.pageLabels.length !== 0;
+    if (isCustomPageLabels) {
+      this.setState({ isCustomPageLabels: true });
+    }
+
+    if (
```
Comment (justinjung04) [inline src/components/PageNavOverlay/PageNavOverlay.js]: Could you put this in one line or make a variable?

File: src/redux/actions/exposedActions.js [hunk 2]
```diff
@@ -72,3 +73,11 @@ export const setActiveLeftPanel = dataElement => (dispatch, getState) => {
 export const setSortNotesBy = sortNotesBy => ({ type: 'SET_SORT_NOTES_BY', payload: { sortNotesBy } });
 export const updateTool = (toolName, properties) => ({ type: 'UPDATE_TOOL', payload: { toolName, properties } });
 export const setCustomPanel = newPanel => ({ type: 'SET_CUSTOM_PANEL', payload: { newPanel } });
+export const setPageLabels = pageLabels => dispatch => {
+  if (pageLabels.length !== core.getTotalPages()) {
+    console.warn('Your custom page labels don not have the same length as the number of pages in this document. This may leads to bugs in some components');
```
Comment (justinjung04) [inline src/redux/actions/exposedActions.js]: I think we can just say "Number of page labels do not match with the total pages." and return

File: src/components/PageNavOverlay/PageNavOverlay.js (hunk 1 above)
Comment (justinjung04) [inline src/components/PageNavOverlay/PageNavOverlay.js]: Could you put this in one line or make a variable?

File: src/redux/actions/exposedActions.js (hunk 2 above)
Comment (ZhijieZhang) [reply src/redux/actions/exposedActions.js]: Yeah this makes more sense, thanks!

**Pull request merged**